### PR TITLE
Validation: Descriptions 

### PR DESCRIPTION
**Pull request by SueADarby**
Third batch of descriptions.

### CODE REVIEW

File: iis/extensions/database-manager-reference/index-ismarkedfordelete-property-microsoft-web-management-databasemanager.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Index.IsMarkedForDelete Property (Microsoft.Web.Management.DatabaseManager)
+description: This page details the syntax for the Index.IsMarkedForDelete property. This property gets or sets a value that indicates whether the index is marked for deletion.
```
Comment (dereklh7) [inline iis/extensions/database-manager-reference/index-ismarkedfordelete-property-microsoft-web-management-databasemanager.md:3]: Description is too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main. Also, remove the "this page" part of the description.

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-videoplaybacktrackproperty-field-microsoft-web-media-smoothstreaming_1.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: SmoothStreamingMediaElement.VideoPlaybackTrackProperty Field (Microsoft.Web.Media.SmoothStreaming)
+description: This article provides code samples and syntax for the SmoothStreamingMediaElement.VideoPlaybackTrackProperty field which represents a dependency property that specifies the VideoPlaybackTrack property.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-videoplaybacktrackproperty-field-microsoft-web-media-smoothstreaming_1.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/extensions/smooth-streaming-client/ssmedrmsetupdecryptorcompletedeventargs-class-microsoft-web-media-smoothstreaming.md [hunk 3]
```diff
@@ -1,10 +1,11 @@
 ---
 title: SSMEDrmSetupDecryptorCompletedEventArgs Class (Microsoft.Web.Media.SmoothStreaming)
+description: This page details the syntax for the SSMEDrmSetupDecryptorCompletedEventArgs class. This class represents event arguments for a DrmSetupDecryptorCompleted event.
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/ssmedrmsetupdecryptorcompletedeventargs-class-microsoft-web-media-smoothstreaming.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/extensions/smooth-streaming-client/ssmedrmsetupdecryptorcompletedeventargs-class-microsoft-web-media-smoothstreaming.md [hunk 4]
```diff
@@ -1,10 +1,11 @@
...
 TOCTitle: SSMEDrmSetupDecryptorCompletedEventArgs Class
 ms:assetid: T:Microsoft.Web.Media.SmoothStreaming.SSMEDrmSetupDecryptorCompletedEventArgs
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.ssmedrmsetupdecryptorcompletedeventargs(v=VS.95)
 ms:contentKeyID: 46307952
-ms.date: 05/31/2012
+ms.date: 05/31/2012git
```
Comment (dereklh7) [inline iis/extensions/smooth-streaming-client/ssmedrmsetupdecryptorcompletedeventargs-class-microsoft-web-media-smoothstreaming.md:8]: ? This looks like a typo from trying to type in Git Bash? Remove the "git" here.

File: iis/extensions/transform-manager/ilogger-writeline-method-microsoft-web-media-transformmanager.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: ILogger.WriteLine Method  (Microsoft.Web.Media.TransformManager)
+description: This article provides information on the WriteLine method overload list. For complete information about this member, including syntax, usage, and examples, click a name in the overload list.
```
Comment (dereklh7) [inline iis/extensions/transform-manager/ilogger-writeline-method-microsoft-web-media-transformmanager.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/wmi-provider/configuredobject-class.md [hunk 6]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ConfiguredObject Class1"
+description: This page details the syntax for the ConfiguredObject Class1. It acts as a base type for namespace objects whose configuration can be specified in configuration sections.
```
Comment (dereklh7) [inline iis/wmi-provider/configuredobject-class.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/wmi-provider/wsdlhelpgeneratorinfo-class.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "WsdlHelpGeneratorInfo Class"
+description: This article specifies the XML Web service Help page (an .aspx file) that is displayed to a browser when the browser navigates directly to an .asmx (XML Web services) page.
```
Comment (dereklh7) [inline iis/wmi-provider/wsdlhelpgeneratorinfo-class.md:3]: Too long per https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-seo-meta-description?branch=main.

File: iis/wmi-provider/httptracingsection-class.md [hunk 8]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "HttpTracingSection Class"
+description: The HttpTracingSection class corresponds to the <httpTracing> section in ApplicationHost.config and is used for Event Tracing.
```
Comment (dereklh7) [inline iis/wmi-provider/httptracingsection-class.md:3]: Remove the brackets around httpTracing - may cause HTML flags.

**Comment by SueADarby** [comment]
**Global effort to fix build validation errors**
@Rick-Anderson  and @mcleblanc 
The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

**Pull request merged**